Core.ApplicationServices/UserService.cs

```diff
@@ -219,5 +225,23 @@ public void Dispose()
         {
             _crypt?.Dispose();
         }
+
+        public Result<IQueryable<User>, OperationError> GetUsersWithCrossAccess()
+        {
+            if(_authorizationContext.GetCrossOrganizationReadAccess() != CrossOrganizationDataReadAccessLevel.All)
+            {
+                return new OperationError(OperationFailure.Forbidden);
+            }
+            return Result<IQueryable<User>, OperationError>.Success(new QueryByApiOrStakeHolderAccess().Apply(_repository.GetUsers()));
```

Comment:
Navnet holdes nok mere clean for forandringer hvis vi navngiver den QueryByCrossOrganizationPermissions